docs/data/toolpad/core/components/all-components.md

```diff
@@ -0,0 +1,5 @@
+# Toolpad components
```

Comment:
I already created something like this at `toolpad/core/all-components`. But we can easily move it and rewords thing, I copied the structure from Base UI but don't have a particular preference.

Reply:
Yep, this was a merge gone bad